### ISSUE TITLE
Whislist: Chat history

### ISSUE THREAD

**Issue by RafaelLinux** [Spring-Chobby/Chobby#250]
There is no history on chat, so when I press "arrow up" key, my last sentences doesn't appear.
This is usefull many times.

**Comment by gajop** [comment]
This should be fairly trivial to do. I would make it simple, and basically keep a list of per-channel messages sent and allow navigation by the up/down arrows.
I would avoid complicating it with semi auto-completion, e.g. "Hello<UP_ARROW>" to complete all messages starting with "Hello", as this really isn't a command panel for that use case to be valid (like for example chonsole).

**Comment by RafaelLinux** [comment]
I agree with you,  "autocompletion" is not necessary , but history it's at
least useful.
